app/repository/user.py

```diff
@@ -1,4 +1,5 @@
 from model.user import User
```

Comment:
Just a question, as you are importing all the models in `model.__init__.py`, why not using just:
```suggestion
from model import User
```

Reply:
FIx